README.md

```diff
@@ -36,53 +36,77 @@ Clone this repository as your new project `git clone git@github.com:tforster/web
+1. Create webproducer.yml. The following example shows multiple formats for data, templates and destination by way of aliases introduced in 0.6.0.
 
    ```yml
-   dataSource:
-     ## GraphQL query
-     type: graphql
-     path: https://graphql.datocms.com/
-     token: ${env:GRAPHQL_API_TOKEN}
+   data: ${env:DATA_ALIAS} # Uses an environment variable to specify an alias
+   templates: template-filesystem # Points to an alias
+   destination: # References the destination object directly
+     type: filesystem
+     path: ./dist
+     archive: false
 
-   templateSource:
-     ## AWS S3
-     type: s3
-     path: s3://wp.somebucket.com/src/${env:STAGE}
-     region: us-east-1
+   ####################
+   # Template Aliases #
+   ####################
 
-     ## Local filesystem
+   template-filesystem:
      type: filesystem
      path: ./src
 
-   destination:
-     ## Local filesystem
+   #######################
+   # Destination Aliases #
+   #######################
+
+   destination-filesystem:
      type: filesystem
      path: ./dist
      archive: false
 
-     ## AWS S3
+   ################
+   # Data Aliases #
+   ################
+
+   data-fs:
+     type: filesystem
+     meta: ./src/data
+
+   data-graphql:
+     type: graphql
+     endpoint: https://graphql.datocms.com/
+     meta: s3://mybucket/${env:STAGE}/data # References an S3 bucket and staged key previx
+     region: ca-central-1 # S3 meta references currently require the region to be provided
+     token: ${env:GRAPHQL_API_TOKEN} # Uses an environment variable to hide sensitive data from Git commits
+
+   data-s3:
      type: s3
-     path: s3://www.somebucket.com
+     meta: s3://mybucket/${env:STAGE}/data
+     region: ca-central-1
+
+   data-get-request: # REST not implemented. Future consideration only.
+     type: get
+     endpoint: https://some-endpoint.com/this/that
+
+   data-sql: # SQL not implemented. Future consideration only.
+     type: sql
+     endpoint: Server=myServerName\myInstanceName;Database=myDataBase;User Id=${env:DB_USERNAME};Password=${env:DB_PASSWORD};
+     meta: ./src/data/meta
+
+   data-mongodb: # Mongo not implemented. Future consideration only.
+     type: mongo
+     endpoint: mongodb://mongodb0.example.com:27017
+     meta: s3://mybucket/stage/data/meta
      region: us-east-1
-     archive: false # default
-     setHttpExtension: false # default
-     acl: public-read # default
-     deleteOnSync: true # default
-     webserver: # optional
-       setContentTypeHeader: true # default
-       cloudFrontDistributionId: EZP3VGJ7GV1RO # optional
-
-    # preview is currently tightly bound to the DatoCMS implementation for specifying draft vs published
-    preview: ${env:PREVIEW}
+   # path: A pointer to the root of the data. E.g. a single file containing the data or a database via  connection string
+   # meta: A filesystem or S3 bucket path containing one or both of a query and transform module. Types graphql, sql and mongo all
```

Comment:
Looking at some old files, I see I used `base` (and to a lesser extent `root`) for the name of the `path` property as noted above.  Then for the `meta` property above, the name `path` was used.  Not sure if this helps us get closer to the terminology we want.